examples/maps/maps.bal

```diff
@@ -21,20 +21,20 @@ function main(string... args) {
     addrMap.postalCode = "00301";
     io:println(addrMap);
 
-    // You can use the 'keys' function in the 'maps' package to get an array of keys.
+    // You can use the `keys()` function of the map to get the keys of the map as an array.
     io:println(addrMap.keys());
 
     // Print the number of keys in the map.
     io:println(lengthof addrMap);
 
-    // You can remove a key using the 'remove' method.
+    // You can remove a key using the `remove()` function.
     var isRemoved = addrMap.remove("postalCode");
     io:println(addrMap);
 
-    // Constrained maps can only contain values of type which the map is constrained to.
+    // Constrained maps can only contain values of the specified constrained type.
```

Comment:
'specified constrained type' sounds unusual to me. Shall we say something like 'Constrained maps can only contain values of the specific types that are constrained." WDYT?

Reply:
How about "Constrained maps can only contain values of the type specified by the type descriptor"? Type descriptor is the part between "<>"